Examples/WhisperAX/WhisperAX.xcodeproj/project.pbxproj

```diff
@@ -901,6 +901,7 @@
 				SUPPORTS_MACCATALYST = NO;
 				SUPPORTS_XR_DESIGNED_FOR_IPHONE_IPAD = YES;
 				SWIFT_EMIT_LOC_STRINGS = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
```

Comment:
is this an intended change?

Reply:
I think it helps with debugging so variables are inaccessible in breakpoints, but shouldn't necessarily hurt since it is the default for debug